testnets/testnets.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Joining Testnet
+order: 2
+---
+
+# Joining Testnets
+
+In this section we will demonstrate how to roll your node and start producing blocks in live testnet environment.
+
+## Setup
+
+First of all make sure you followed the installation steps in [build requirements section](./build-requirements.md). You should have the required binaries. If you just want to copy and execute the scripts below, make sure to set up environment variables:
+
+```sh
+export CHAIN_ID=testing
+export TESTNET_NAME=demo-09
+export RPC=rpc.demo-09.cosmwasm.com:443
+export FAUCET=faucet.demo-09.cosmwasm.com
+export SEED_NODE=p2p.demo-09.cosmwasm.com:26656
+```
+
+Also install `jq`.
+
+## Initialize your wallet
+
+Initialize `wasmcli` and generate validator account:
+
+```sh
+wasmcli config chain-id $CHAIN_ID
+wasmcli config trust-node true
+wasmcli config node https://$RPC
+wasmcli config output json
+wasmcli config indent true
+# this is important, so the cli returns after the tx is in a block,
+# and subsequent queries return the proper results
+wasmcli config broadcast-mode block
+
+# check you can connect
+wasmcli query supply total
+wasmcli query staking validators
+wasmcli query wasm list-code
+
+# create wallet
+wasmcli keys add mywallet
+```
+
+In order to join the network as validator, you need some staking tokens. We have a faucet running for that.
+
+Request some tokens from faucet:
+
+```sh
+# copy the address and paste it to ${WALLET_ADDR} below
+wasmcli keys show -a mywallet
+
+# get some hot stake
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"ticker":"STAKE","address":"${WALLET_ADDR}"}' \
+  https://$FAUCET/credit
+
+# get coins for paying fees
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"ticker":"COSM","address":"${WALLET_ADDR}"}' \
+  https://$FAUCET/credit
+```
+
+## Run wasmd node
+
+TODO persistent peer and seed node?
```

Comment:
I've provided how to use a seed node. Not sure if I should provide persistent peer usage as well?

Reply:
Sounds good as seed node. I guess that is more standard. No need to do both. Experts already know how